src/main/java/com/booster/server/NoteService.java

```diff
@@ -12,11 +17,20 @@ public class NoteService {
     public NoteDto create(CreateNoteInput input) {
         var noteEntity = new NoteEntity();
         noteEntity.setContent(input.getContent());
+        noteEntity.setLastSeenAt(LocalDateTime.of(LocalDate.EPOCH, LocalTime.MIN));
 
         noteEntity = noteRepository.save(noteEntity);
 
         return new NoteDto()
                 .setContent(noteEntity.getContent());
     }
 
+    public void updateLastSeenAt(UpdateLastSeenAtInput input) {
+        List<NoteEntity> updatedNoteEntities = noteRepository.findAllById(input.getIds())
+                .stream()
+                .map(note -> note.setLastSeenAt(input.getLocalDateTime()))
+                .toList();
+        noteRepository.saveAll(updatedNoteEntities);
```

Comment:
I am wondering if we need a `Stream` roundtrip here at all.

What if we do smth like this:
```java
List<NoteEntity> notes = noteRepository.findAllById(input.getIds());
notes.forEach(note -> note.setLastSeenAt(input.getLocalDateTime()));
noteRepository.saveAll(notes);
```
This way we make the operation more performant.